### PR TITLE
Tarantool 1.6.8-525

### PR DESCRIPTION
**Pull request by bigbes**
## All Submissions:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

### CODE REVIEW

File: Library/Formula/tarantool.rb [hunk 1]
```diff
@@ -19,22 +19,12 @@ def install
     args = std_cmake_args
     args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
     args << "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
+    args << "-DCMAKE_INSTALL_LOCALSTATEDIR='#{var}'"
```
Comment (DomT4) [inline Library/Formula/tarantool.rb]: Why is `'#{var}'` being passed in `''` whereas the rest are passed directly, out of interest?

Comment (bigbes) [inline Library/Formula/tarantool.rb]: It was a misprint, but it won't affect anything, so I'd forget about it.

Comment (DomT4) [inline Library/Formula/tarantool.rb]: I'd rather we fix it. Someone will spot the inconsistency and try to fix it either by removing the `''` or adding them across the other options, and I'll have likely forgot this conversation by the time that happens.

File: Library/Formula/tarantool.rb [hunk 2]
```diff
@@ -19,22 +19,12 @@ def install
...
+    args << "-DENABLE_DIST=ON"
 
     system "cmake", ".", *args
     system "make"
     system "make", "install"
-
-    inreplace bin/"tarantoolctl", %r{\/usr\/local\/etc\/tarantool\/default\/tarantool}, "#{etc}/default/tarantool"
```
Comment (DomT4) [inline Library/Formula/tarantool.rb]: This has been fixed upstream?

File: Library/Formula/tarantool.rb (hunk 1 above)
Comment (bigbes) [inline Library/Formula/tarantool.rb]: Ok, you're right, I'll do it.

File: Library/Formula/tarantool.rb (hunk 2 above)
Comment (bigbes) [inline Library/Formula/tarantool.rb]: Yes, we've fixed It. `cmake` now configures it.

**Comment by bigbes** [comment]
@DomT4 I've updated brewspec.

File: Library/Formula/tarantool.rb (hunk 2 above)
Comment (DomT4) [inline Library/Formula/tarantool.rb]: Neat, Thanks!

**Comment by bigbes** [comment]
@DomT4 or @mikemcquaid can I do anything to merge this PR?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/cd22791a0f4d9515dc47579d9c66410072c7edd8. Thank you for another contribution to Homebrew @bigbes! :smiley_cat:

**Comment by bigbes** [comment]
@DomT4 Thank you for merging!

**Pull request closed without merging**